quickstarts/202-coe-starter-kit/main.tf

```diff
@@ -0,0 +1,167 @@
+      admin_host_domains                                          = var.core_components_parameters.admin_host_domains,
+      admin_inventory_and_telemetry_in_azure_data_storage_account = var.core_components_parameters.admin_inventory_and_telemetry_in_azure_data_storage_account,
+      admin_inventory_filter_days_to_look_back                    = var.core_components_parameters.admin_inventory_filter_days_to_look_back,
+      admin_is_full_tenant_inventory                              = var.core_components_parameters.admin_is_full_tenant_inventory,
+      admin_maker_dataflow_id                                     = var.core_components_parameters.admin_maker_dataflow_id,
+      admin_model_app_dataflow_id                                 = var.core_components_parameters.admin_model_app_dataflow_id,
+      admin_power_app_environment_variable                        = var.core_components_parameters.admin_power_app_environment_variable,
+      admin_power_app_player_environment_variable                 = var.core_components_parameters.admin_power_app_player_environment_variable,
+      admin_power_automate_environment_variable                   = var.core_components_parameters.admin_power_automate_environment_variable,
+      admin_power_platform_make_security_group                    = var.core_components_parameters.admin_power_platform_make_security_group,
+      admin_power_platform_user_group_id                          = var.core_components_parameters.admin_power_platform_user_group_id,
+      admin_production_environment                                = var.core_components_parameters.admin_production_environment,
+      admin_sync_flow_errors_delete_after_x_days                  = var.core_components_parameters.admin_sync_flow_errors_delete_after_x_days,
+      admin_user_photos_forbidden_by_policy                       = var.core_components_parameters.admin_user_photos_forbidden_by_policy,
+      coe_environment_request_admin_app_url                       = var.core_components_parameters.coe_environment_request_admin_app_url,
+    }
+  }
+}
+
+# //create coe-kit environment
+resource "powerplatform_environment" "coe_kit_env" {
+  provider         = powerplatform.pp
+  location         = var.environment_parameters.env_location
+  display_name     = var.environment_parameters.env_name
+  environment_type = var.environment_parameters.env_type
+  dataverse = {
+    language_code     = 1033
+    currency_code     = "USD"
+    security_group_id = "00000000-0000-0000-0000-000000000000"
+  }
+}
+
+//install creator-kit-core solution
+resource "powerplatform_solution" "creator_kit_solution_install" {
+  provider       = powerplatform.pp
+  environment_id = powerplatform_environment.coe_kit_env.id
+  solution_file  = module.creator_kit.creator_kit_core_solution_zip_path
+  solution_name  = "CreatorKitCore"
+}
+
+module "create_connections" {
```

Comment:
this module executes every time, but it should execute only when connection creation mode is set to `test_engine`